app/views/vacancies/_filters.html.haml

```diff
@@ -1,19 +1,19 @@
 - content_for :advanced_filters do
   .govuk-form-group
     = label_tag 'radius', t('jobs.filters.radius'), class: 'govuk-label'
-    = select_tag 'radius', options_for_select(radius_filter_options, radius), class: 'govuk-select form-control form-control-block govuk-!-width-full'
```

Comment:
Super minor, think this and the previous commit make more sense together as one commit.  I.e. removing both the class and where it's used.

Reply:
Will squash it with the previous commit with a rebase

Reply:
squashed into https://github.com/DFE-Digital/teacher-vacancy-service/pull/1057/commits/c28f9e43959456e9bea5a5b3fd9c2fb9a90c3c0d